src/tests/moods.js

```diff
@@ -0,0 +1,37 @@
+const add = require("../actions/moods/add-moods"),
+	remove = require("../actions/moods/remove-moods"),
+	specificMoods = require("../actions/moods/specific-moods"),
+	updateMoods = require("../actions/moods/update-moods"),
+	mood = require("../models").mood;
+
+describe("Moods", () => {
+	beforeAll(() => {
+		mood.create({
+			name: "Vis"
+		});
+	});
+	it("Add a mood.", () => {
```

Comment:
`it("can add a mood"`. This reads much better, and gives a neat overview in the output of testing.